src/Codeception/Test/Loader/Unit.php

```diff
@@ -59,14 +71,15 @@ protected function createTestFromPhpUnitMethod(\ReflectionClass $class, \Reflect
         return $test;
     }
 
-    protected function enhancePhpunitTest(\PHPUnit\Framework\Test $test)
+    protected function enhancePhpunitTest(Test $test): void
```

Comment:
There are so many classes named `Test` that it would be good for readability if this alias was `PHPUnitTest`